src/site/content/en/blog/http-cache-security/index.md

```diff
@@ -0,0 +1,180 @@
+---
+layout: post
+title: Improve security and privacy by updating HTTP Cache
+authors:
+  - arthursonzogni
+description: Forgetting or misusing the Cache-Control header might negatively impact the security of your website and your users' privacy. Get recommendations for high-value websites.
+date: 2022-02-07
+thumbnail: image/VbsHyyQopiec0718rMq2kTE1hke2/3MnHBfutFgu2ai1F2mUH.jpg
+alt: Two ducks move through the water, creating a rippling wave.
+tags:
+  - blog
+  - security
+---
+
+By default, resources are always allowed to be cached by any type of cache.
+Not using or misusing the `Cache-Control` header might negatively impact the
+security of your website and your users' privacy.
+
+For personalized responses you want to keep private, we recommend you either:
+
+*  Set an appropriate [secondary cache
+   key](https://httpwg.org/specs/rfc7234.html#caching.negotiated.responses).
+   If the response depends on cookies, then set `Vary: Cookie`.
+*  Prevent intermediaries from caching the resource. Set
+   `Cache-Control: private`.
```

Comment:
@JackJey suggested putting the emphasis on the second point and making it clearer on the first point that `Vary:Cookie` can be used only when the Cookie is the credentials.

Maybe something like:
```
*  Prevent intermediaries from caching the resource. Set
   `Cache-Control: private`.
*  Set an appropriate [secondary cache
   key](https://httpwg.org/specs/rfc7234.html#caching.negotiated.responses).
   If the response varies upon cookie, because it stores credentials, then set `Vary: Cookie`.
```

Here and in L150.

WDYT @JackJey ?

Reply:
I've updated as follows: 

> *  Prevent intermediaries from caching the resource. Set `Cache-Control: private`.
> *  Set an appropriate [secondary cache key]https://httpwg.org/specs/rfc7234.html#caching.negotiated.responses). If the response varies due to cookies&mdash;which can happen when the cookie stores credentials&mdash;then set `Vary: Cookie`.